src/kaocha/plugin/junit_xml.clj

```diff
@@ -100,32 +100,35 @@
                         (hierarchy/fail-type? m) (failure->xml m)))
                     events)}))
 
-(defn suite->xml [suite index]
+(defn suite->xml [{:keys [no-system-out?]} suite index]
   (let [id (::testable/id suite)]
-    {:tag :testsuite
-     :attrs (-> {:name (test-name suite)
-                 :id index
-                 :hostname "localhost"}
-                (merge (stats suite) (time-stat suite))
-                (assoc :package (if (qualified-ident? id)
-                                  (namespace id)
-                                  "")))
+    {:tag     :testsuite
```

Comment:
Small tip for the future, avoid making unnecessary whitespace changes when submitting a patch, it makes it much easier to review.